CHANGELOG.adoc

```diff
@@ -13,6 +13,25 @@
 
 == Unreleased changes (release date TBD)
 
+Breaking change: this release removes the `ToString` implementation for
+`TypeSpace` and the `rustfmt-wrapper` dependency. This allows users to use a
+pretty printing method of choice. Users can use the `ToTokens` implementation of
+`TypeSpace`:
+
+Before:
+[source,rust]
+----
+let typespace = TypeSpace::default();
+typespace.to_string()
+----
+
+After:
+[source,rust]
+----
+let typespace = TypeSpace::default();
+typespace.to_stream().to_string()
```

Comment:
I can amend this when I cut the release (soon!) but I'd like a little more detail about e.g. to link a doc section on formatting